python/TestHarness/TestHarness.py

```diff
@@ -127,10 +129,13 @@ def __init__(self, argv, app_name, moose_dir):
                (self.options.cli_args.find('--parallel-mesh') != -1 or self.options.cli_args.find('--distributed-mesh') != -1)):
 
             option_set = set(['ALL', 'PARALLEL'])
-            self.checks['mesh_mode'] = option_set
+            checks['mesh_mode'] = option_set
 
         method = set(['ALL', self.options.method.upper()])
-        self.checks['method'] = method
+        checks['method'] = method
+
+        # This is so we can easily pass checks around to any scheduler plugin
+        self.options._checks = checks
```

Comment:
This was to clean up the requirements as we pass the tester around. When ever we perform a checkRunnable, we needed to pass checks, failed_tests, and options.

Now we just pass options.